doc/akri_connector/connector-metadata-schema.json

```diff
@@ -1,13 +1,139 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "aioConnectorMetadataSchemaVersion": "2.0-preview",
+  "aioConnectorMetadataSchemaVersion": "3.0-preview",
+  "definitions": {
+    "modelLimits": {
+      "type": "object",
+      "description": "Describes the limits to this model type that this connector supports. For instance, a connector may support between 0 to 5 datasets.",
+      "properties": {
+        "maximum": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The maximum number of this model type that this connector supports. If not specified, the default is whatever limits the ADR service itself has. If this value is 0, then this connector does not support using this model type (i.e. the connector supports datasets, but not events)."
```

Comment:
The above says that the `maximum` property has a `minimum` of 1. The description mentions that the maximum could be 0, is that allowed?